app/main.py

```diff
@@ -1,25 +1,79 @@
-from abc import ABC
+from abc import ABC, abstractmethod
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: int, owner: str) -> tuple:
+        return self.min_amount, self.max_amount
+
+    def __set__(self, instance: int, value: int) -> None:
+        raise AttributeError("Cannot set value directly")
+
+    def __set_name__(self, owner: str, name: str) -> None:
+        self.name = name
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(
+            self,
+            age: IntegerRange,
+            weight: IntegerRange,
+            height: IntegerRange
+    ) -> None:
+        self.age_range = age
+        self.weight_range = weight
+        self.height_range = height
+
+    @abstractmethod
+    def can_access(self, visitor: Visitor) -> bool:
+        pass
```

Comment:
Do not create this abstract method, `can_access` should be present only in the `Slide` class.